binding_generator.py

```diff
@@ -454,6 +454,8 @@ def generate_builtin_class_header(builtin_api, size, used_classes, fully_used_cl
     result.append("")
     result.append("\tstatic struct _MethodBindings {")
 
+    result.append(f"\t\tGDExtensionTypeFromVariantConstructorFunc from_variant_constructor;")
```

Comment:
Is there a need to make this an f-string?  I don't see any variable substitution.

Reply:
Ah, thanks, you're right, it doesn't need to be an f-string, probably introduced by copy-paste. Fixed in my latest push :-)